pkg/bootstrapper/bootstrapper_test.go

```diff
@@ -253,32 +253,6 @@ func TestCloudConfNotPresent(t *testing.T) {
 	assert.False(t, present, "cloud-config option is not present in kubelet args")
 }
 
-// TestKubeletArgs tests if proper kubelet arguments are present after parsing the ignition file
-func TestKubeletArgs(t *testing.T) {
```

Comment:
just to know why is this test case removed..?

Reply:
Please call this out in the commit message @ravisantoshgudimetla

Reply:
I was using this to pass check for the kubelet args obtained after parsing ignition file, specifically nodelabels which I am not doing anymore. I updated the commit message.